sqlitecluster/SQLiteNode.cpp

```diff
@@ -1278,7 +1278,21 @@ void SQLiteNode::_onMESSAGE(SQLitePeer* peer, const SData& message) {
             peer->commandAddress = message["commandAddress"];
         }
         peer->setCommit(message.calcU64("CommitCount"), message["Hash"]);
-
+        
+        // We check the commit difference with 12,500 commits behind because that 
+        // represents ~30s of commits. If we're behind, let's close the command port
+        // so we can catch up with the cluster before processing new commands.
+        const string blockReason = "COMMITS_LAGGING_BEHIND";
+        const int64_t currentCommitDifference =  peer->commitCount - getCommitCount();
+        if (peer == _leadPeer && currentCommitDifference >= 12'500 && !_blockedCommandPort) {
+            SINFO("Node is lagging behind, closing command port so it can catch up.");
+            _server.blockCommandPort(blockReason);
+            _blockedCommandPort = true;
+        } else if (currentCommitDifference < 1'000 && _blockedCommandPort) {
```

Comment:
The PR descriptions says 50K / 10K, but the diff shows 12.5K / 1K. Can we add a comment explaining the reasoning behind the 1K? The one for 12.5K is good because it says it's roughly 30s of commits

Reply:
Changed the PR description, added comment.